README.md

```diff
@@ -10,17 +10,31 @@ Test via ```nosetests```.
 ### Usage: 
     POST   on /shopcarts: creates new shopcart.
         Expects JSON body data:
-            item_id: -1
+            item_id: 0
             user_id: int
+            item_name: string, "ring" + str(item_id)
+            quantity: int, range from 0 to 999
+            price: float, range from 0.0 to 0.1
+    GET    on /shopcarts: returns all of the shopcarts.
+    GET    on /shopcarts/<user-id>: returns items in <user-id> shopcart.
+    DELETE on /shopcarts/<user-id>: deletes <user-id> shopcart.
+    POST   on /shopcarts/<user-id>/items: creates new item in <user-id> shopcart.
+        Expects JSON body data:
+            item_id: int
+            quantity: int 
             item_name: string
-            quantity: int
             price: float
```

Comment:
This should be int or float

Reply:
Also this route will return the newly created item as JSON

Reply:
Thanks for your review. Just about the type of price. I am not sure about its specific type. As mentioned in the Class Shopcart in model.py, the price is defined as Float.
I will fix other problems later. Thanks Timothy.

Reply:
Yes in our database it is stored as float; I just mean it's okay to pass in say 2 as price since it's JSON which is text any way. Yeah actually maybe you can just leave it as float since that can also mean 2 is okay right?

Reply:
Got it. Fixed.